src/mesh/region.cc

```diff
@@ -146,6 +146,12 @@ Region RegionDB::add_region(unsigned int id, const std::string &label) {
         THROW(ExcNonuniqueLabel() << EI_Label(label) << EI_ID(id) << EI_IDOfOtherLabel(it_label->get_id()) );
     }
 
+    OnlyIDIter it_only_id = region_set_.get<OnlyID>().find(id);
```

Comment:
region_set_ is RegionTable (database of all region) - variable name is misleading.
I propose to rename region_set_ (e.g. to region_table_).

Method is renamed.